flavors/standard-Exasol-8-cuda-ml/flavor_customization/Dockerfile

```diff
@@ -0,0 +1,76 @@
+############################################################################################
```

Comment:
Do we actually need this. Probably not. This flavor is not for production use.

Reply:
I see. I will remove it